<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dopusk02_TNR</class>
 <widget class="QMainWindow" name="Dopusk02_TNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dopusk02_TNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="EditPanel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="LabelEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ColorPicker">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>82</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SetButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 40pt;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Hello</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>500</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	width: 100%;	
	background-color:  rgb(210, 230, 255);
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <widget class="QPushButton" name="EditButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ByeButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Bye</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
